### ISSUE TITLE
Torch wheel and libmpi_cxx.so.40

### ISSUE THREAD

**Issue by Lunatik00** [xuhuisheng/rocm-gfx803#4]
The Dockerfile of rocm 4.5 full (rocm/dev-ubuntu-20.04:4.5-complete) works with the tensorflow wheel for python 3.8, but i have been unable to make it work with torch

![image](https://user-images.githubusercontent.com/72462823/147153476-c34b53f5-76f9-457b-8513-ff47c307dd29.png)

I have searched but the solutions i get are mostly out of date and i have not been able to solve the issue by myself

**Comment by xuhuisheng** [comment]
You can try this:
```
sudo apt install -y liblmdb-dev libopencv-highgui-dev libopencv-contrib-dev libopenblas-dev
```

**Comment by Lunatik00** [comment]
solved in #7 it should be added to the instructions

**Comment by xuhuisheng** [comment]
This issue didn't update one years ago.
I will close it, please re-open if needed.
